packages/modelviewer.dev/examples/lazy-loading.html

```diff
@@ -81,6 +81,11 @@ <h2 class="demo-title">Display a poster until loaded</h2>
           <h4></h4>
         </div>
         <example-snippet stamp-to="demo-container-1" highlight-as="html">
+          <style>
+            model-viewer#reveal {
+              --poster-color: transparent;
+            }
+          </style>
```

Comment:
I think we should move this inside the template to make it a visible part of our documentation. We should also add this to the example below. The third example seems to already work without needing this, even though it is also a transparent poster. Do you know why?

Reply:
Oh, sorry. This was a mistake. It should be in the template.

Reply:
The third example replaces the default poster (via a `slot`-ed alternative), so `--poster-color` no longer factors into the equation.

Reply:
I think we still want to add this change to the second example, as it is like the first.